spec/manual/https_client_spec.cr

```diff
@@ -12,4 +12,8 @@ describe "https requests" do
       break
     end
   end
+
+  it "can fetch from IIS servers that don't shutdown gracefully" do
+    HTTP::Client.post("https://login.microsoftonline.com/common/oauth2/token", form: {"hi" => "a"}).body
```

Comment:
This imo shouldn't be added - we don't want to rely on 3rd party services in tests.

Reply:
This is not necessary since we can reproduce the behaviour in a regular spec.